aws/resource_aws_network_acl_association_test.go

```diff
@@ -0,0 +1,69 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSNetworkAclAssociation(t *testing.T) {
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:      func() { testAccPreCheck(t) },
+		IDRefreshName: "aws_network_acl.bar",
+		Providers:     testAccProviders,
+		CheckDestroy:  testAccCheckAWSNetworkAclDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSNetworkAclAssoc,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testCheckAwsRMNetworkAclAssocExists("aws_network_acl_association.test"),
+				),
+			},
+		},
+	})
+}
+
+func testCheckAwsRMNetworkAclAssocExists(name string) resource.TestCheckFunc {
```

Comment:
This method should check that the association exists, by requesting AWS.
Do you think you could get some inspiration from [testAccCheckAWSNetworkAclExists](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_network_acl_test.go#L359-L385)?